wolfssl/openssl/dh.h

```diff
@@ -14,12 +14,13 @@
 struct WOLFSSL_DH {
     WOLFSSL_BIGNUM* p;
     WOLFSSL_BIGNUM* g;
+    WOLFSSL_BIGNUM* q;
```

Comment:
Shouldn't q be set to NULL in InitwolfSSL_DH() and wolfSSL_BN_free()d in wolfSSL_DH_free() in case the user sets it even if we aren't currently using it?